lib/before-prepareJS.js

```diff
@@ -71,8 +7,9 @@ module.exports = function ($mobileHelper, $projectData, hookArgs) {
 	const config = {
 		env,
 		platform,
-		bundle: appFilesUpdaterOptions.bundle
+		bundle: appFilesUpdaterOptions.bundle,
+		watch: false
```

Comment:
We are preparing the nativescript-dev-webpack for watch but it is not there yet.